docs/solutions/k8s-on-dcos.md

```diff
@@ -0,0 +1,271 @@
+# Blueprint - Kubernetes on the Mesosphere DCOS
+
```

Comment:
In the second diagram, I was a little surprised to see "Docker/cgroups" listed as an app. I can see why you might do that, but an alternative would be to put it between "DCOS kernel" and "(virtual) machine." You could put it directly above some subset of the nodes, to hint that some nodes can be running containers at the same time as others aren't.